packages/dashboard-frontend/src/store/SanityCheck/index.ts

```diff
@@ -118,6 +118,12 @@ export const actionCreators: ActionCreators = {
           type: Type.RECEIVED_BACKEND_CHECK_ERROR,
           error: errorMessage,
         });
+        if (e.message) {
+          console.error(e.message);
+        }
+        if (e.response && e.response.data && e.response.data.trace) {
+          console.error(e.response.data.trace.join('\n'));
```

Comment:
```suggestion
          console.error(common.helpers.errors.getMessage(e));
```

@vinokurig Could you move the next code  `e.response.data.trace.join('\n')` into 'getMessage'  function (https://github.com/eclipse-che/che-dashboard/blob/main/packages/common/src/helpers/errors.ts#L63-L65) ?

Reply:
This is a stack trace list and it is provision error specific, so it makes no sense to put it in ti the common error class.